news/6404.feature

```diff
@@ -0,0 +1,2 @@
+Support ``--target`` installations with ``pip freeze`` with the ``--path``
+argument.
```

Comment:
Here is a slight rewording that I think might be better: "Add a `--path` argument to `pip freeze` to support `--target` installations."